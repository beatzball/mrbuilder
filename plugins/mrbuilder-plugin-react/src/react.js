const React     = require('react-internal');
const PropTypes = require('prop-types');

React.PropTypes = PropTypes;
if (!React.createClass) {
    //make lazy to prevent cyclic dependency.
    React.createClass = (...v) => (React.createClass = require('create-react-class'))(...v);

}
module.exports = React;
