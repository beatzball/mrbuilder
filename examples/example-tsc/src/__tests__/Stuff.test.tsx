import * as React from 'react';
import Stuff from '../index';
import {create} from 'react-test-renderer';

test('Should render lib', () => {
    const app = create(<Stuff name={'stuff'}/>);
    expect(app.toJSON()).toMatchSnapshot();

});
