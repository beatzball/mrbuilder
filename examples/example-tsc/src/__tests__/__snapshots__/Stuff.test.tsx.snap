// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render lib 1`] = `
<span>
  stuff
</span>
`;
